k8s/trillian/manifest/logserver.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: $NAME-logserver-service
+    labels: &LogServerLabels
+      app.kubernetes.io/name: "$NAME"
+      app.kubernetes.io/component: logserver
+    annotations:
+        cloud.google.com/load-balancer-type: "Internal"
+  spec:
+    ports:
+    - name: grpclb
+      port: 8090
+    - name: metrics
+      port: 8091
+    selector: *LogServerLabels
+    type: LoadBalancer
+- apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: $NAME-logserver-deployment
+    labels: *LogServerLabels
+  spec:
+    replicas: $TRILLIAN_LOGSERVER_REPLICAS
+    selector:
+      matchLabels: *LogServerLabels
+    template:
+      metadata:
+        labels: *LogServerLabels
+      spec:
+        restartPolicy: Always
+        containers:
+        - name: trillian-logserver
+          args: [
+          "$(STORAGE_FLAG)",
+          "--storage_system=$(STORAGE_SYSTEM)",
+          "--quota_system=etcd",
+          "--etcd_servers=$NAME-etcd-cluster-client:2379",
+          "--etcd_http_service=$NAME-logserver-http",
```

Comment:
Is this name correct? It is not defined in this PR.

Reply:
This is not a Kubernetes service. It is a prefix added to some Etcd keys used by this binary.